static/app/views/replays/replays.tsx

```diff
@@ -35,7 +36,17 @@ class Replays extends React.Component<Props> {
       id: '',
       name: '',
       version: 2,
-      fields: ['eventID', 'timestamp', 'replayId', 'user'],
+      fields: [
+        'eventID',
+        'timestamp',
+        'replayId',
+        'user.display',
+        'user.id',
+        'user.ip',
+        'user.email',
+        'user.username',
```

Comment:
Hmm, I don't love that we have to do this in order to reconstruct a user object and to force it to work with `<UserBadge>`. `user.display` seems to return `<type>:<value>`, which is really all we need.

I'm thinking we merge this PR as-is as it's a big improvement in the UI, but follow-up to clean this up a bit? Thoughts @Danondso?

Reply:
I can tweak it now to clean it up, just need to do:
```javascipt
      fields: ['eventID', 'timestamp', 'replayId', 'user.display', 'url'],
```
and then update the user prop object right?
```javascript
              user={{
                username: replay['user.display'],
                id: replay['user.display'],
                ip_address: replay['user.display'],
                name: replay['user.display'],
                email: replay['user.display'],
              }}
```
I wasn't sure if I'm stuck having to make that entire object to get it to work so if I can reduce that more I'm game for it. 

But if more work is needed then I'm fine with revisiting it.

Reply:
Oh yeah, using only `user.display` like that would also work... let's do that for now

Reply:
Now that I think about it, `UserBadge` was intended for *internal* (to Sentry) users, and not external users like this, but I think this is good. Let's leave the bold in as I'd rather follow our existing component designs rather than customize them to exactly match design mocks.